pkg/plaid_cli/linker.go

```diff
@@ -125,47 +169,31 @@ func handleLink(linker *Linker) func(w http.ResponseWriter, r *http.Request) {
 }
 
 type LinkTmplData struct {
-	PublicKey   string
-	Environment string
```

Comment:
I couldn't find anywhere in the new link flow Plaid docs that mentioned specifying which environment you are in. Maybe this is now implied by the client credentials?